metron-platform/metron-elasticsearch/src/test/java/org/apache/metron/elasticsearch/integration/ElasticsearchMetaAlertIntegrationTest.java

```diff
@@ -83,6 +90,7 @@ public static void setup() throws Exception {
         put("es.date.format", DATE_FORMAT);
       }
     };
+    accessConfig.setMaxSearchResults(1000);
```

Comment:
What's the reason for adding this to the test?

Reply:
This is necessary or it won't work.  There is a check in ElasticsearchDao.search(SearchRequest searchRequest, QueryBuilder queryBuilder) that calls accessConfig.getMaxSearchResults().

Reply:
Makes sense. Do you know if there's a reason we don't default there if it's not set? It's outside the scope of this, so if you don't know I'm not worried, but it seems like it's an opportunity.

Reply:
I'm not sure but adding that should be trivial.  The REST layer does provide a default but I can add it here too.

Reply:
I wouldn't worry about it for this PR, just something to think about.